tsconfig.json

```diff
@@ -12,7 +12,7 @@
     "disableSourceOfProjectReferenceRedirect": false  /* Disable preferring source files instead of declaration files when referencing composite projects */,
 
     /* Language and Environment */
-    "target": "es6"                                /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */,
+    "target": "ES2020"                                /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */,
```

Comment:
needed to bump the target to support the named groups in regex

this was likely due to a bump in the TS version and the compiler being more strict